jupyter_server/services/sessions/sessionmanager.py

```diff
@@ -243,14 +249,26 @@ async def session_exists(self, path):
                 exists = True
         return exists
 
-    def new_session_id(self):
+    def new_session_id(self) -> str:
         "Create a uuid for a new session"
         return str(uuid.uuid4())
 
     async def create_session(
-        self, path=None, name=None, type=None, kernel_name=None, kernel_id=None
-    ):
-        """Creates a session and returns its model"""
+        self,
+        path: Optional[str] = None,
+        name: Optional["ModelName"] = None,
```

Comment:
`ModelName` should not be quoted here.
```suggestion
        name: Optional[ModelName] = None,
```